_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,132 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+### Expectation
+
+Each target element is [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by an element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has a [semantic role](#semantic-role) matching one of the [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+## Assumptions
+
+_There are currently no assumptions_
+
+## Accessibility Support
+
+This rule relies on assistive technologies to recognize owned elements, also when they are nested descendants that are not immediate children. Certain assistive technologies cannot recognize owned elements that not immediate children, unless workarounds are used.
```

Comment:
Suggested edit: "This rule relies on assistive technologies to recognize owned elements, as defined by [WAI-ARIA](https://www.w3.org/TR/wai-aria), This includes when they are nested descendants that are not immediate children. However, some assistive technologies do not recognize owned elements that are not immediate children, unless workarounds are used."